### PR TITLE
[MRG] replace brainomics examples with neurovault one

### PR DESCRIPTION
**Pull request by bthirion**
Note that we lose
* the possibilty to plot on individual anatomy (not available in Neurovault)
* side variables to better showcase grup analyses (#1708 )

### CODE REVIEW

**Comment by jeromedockes** [comment]
would it be better to have a one-line wrapper function in datasets, with a name like "fetch_demo_localizer" or "fetch_example_stat_map", that makes the call to fetch_neurovault_ids and use it in the examples? It may look less scary in the examples and avoid remembering the hard-coded neurovault id when writing new examples in the future

**Comment by GaelVaroquaux** [comment]
Yes indeed. The goal is to provide very simple examples. 

⁣Sent from my phone. Please forgive typos and briefness.​

**Comment by GaelVaroquaux** [comment]
Merging master in this branch. Beware, I am pushing to your branch

File: examples/01_plotting/plot_demo_glass_brain_extensive.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 :ref:`Section 4.3 <display_modules>` for more details about display objects
 in Nilearn.
 
-Also, see :func:`nilearn.datasets.fetch_localizer_button_task` for details
-about the plotting data and its experiments.
+Also, see :func:`nilearn.datasets.fetch_neurovault_ids` for details
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_demo_glass_brain_extensive.py]: It should be fetch_neurovault_motor_task here.

File: examples/01_plotting/plot_demo_glass_brain_extensive.py [hunk 2]
```diff
@@ -30,45 +30,41 @@
...
-tmap_filenames = datasets.fetch_localizer_button_task()['tmaps']
-print(tmap_filenames)
-
-###############################################################################
-# tmap_filenames is returned as a list. We need to take first one
-tmap_filename = tmap_filenames[0]
+# Let us now retrieve a motor task contrast map
+# corresponding to a group one-sample t-test
 
+motor_images = datasets.fetch_neurovault_motor_task()
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_demo_glass_brain_extensive.py]: We should probably print the value of motor_images here, as it used to be done. It's didactic for users.

File: examples/04_manipulating_images/plot_extract_rois_statistical_maps.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 This example shows how to extract regions or separate the regions
 from a statistical map.
 
-We use localizer t-statistic maps from :func:`nilearn.datasets.fetch_localizer_contrasts`
+We use localizer t-statistic maps from :func:`nilearn.datasets.fetch_neurovault`
```
Comment (GaelVaroquaux) [inline examples/04_manipulating_images/plot_extract_rois_statistical_maps.py]: We should have fetch_neurovault_motor_task here

File: examples/04_manipulating_images/plot_resample_to_template.py [hunk 4]
```diff
@@ -10,34 +10,32 @@
 
 ###############################################################################
 # First we load the required datasets using the nilearn datasets module.
-from nilearn.datasets import fetch_localizer_button_task
+from nilearn.datasets import fetch_neurovault_ids
```
Comment (GaelVaroquaux) [inline examples/04_manipulating_images/plot_resample_to_template.py]: Why are we important fetch_neurovault_ids and using fetch_neurovault_motor_task below?

File: nilearn/datasets/__init__.py [hunk 5]
```diff
@@ -27,7 +27,11 @@
...
 from .utils import get_data_dirs
-from .neurovault import fetch_neurovault, fetch_neurovault_ids
+from .neurovault import (fetch_neurovault,
+                         fetch_neurovault_ids,
+                         fetch_neurovault_motor_task,
+                         fetch_neurovault_auditory_computation_task
+)
 
 __all__ = ['MNI152_FILE_PATH', 'fetch_icbm152_2009', 'load_mni152_template',
            'fetch_oasis_vbm',
```
Comment (GaelVaroquaux) [inline nilearn/datasets/__init__.py]: We need to add those functions to the "__all__"

File: nilearn/datasets/neurovault.py [hunk 6]
```diff
@@ -2600,3 +2600,86 @@ def fetch_neurovault_ids(
...
+    ------
+
+    This function is only a caller for the fetch_localizer_contrasts in order
+    to simplify examples reading and understanding.
+    The 'left vs right button press' contrast is used.
+
+    See Also
+    ---------
+    nilearn.datasets.fetch_neurovault_ids
+    nilearn.datasets.fetch_neurovault
```
Comment (GaelVaroquaux) [inline nilearn/datasets/neurovault.py]: Maybe also fetch_neurovault_auditory_computation_task in the seealso

File: nilearn/datasets/neurovault.py [hunk 7]
```diff
@@ -2600,3 +2600,86 @@ def fetch_neurovault_ids(
...
```
Comment (GaelVaroquaux) [inline nilearn/datasets/neurovault.py]: Maybe also fetch_neurovault_motor_task in the seealso

File: nilearn/datasets/neurovault.py [hunk 8]
```diff
@@ -2600,3 +2600,86 @@ def fetch_neurovault_ids(
...
+              dictionaries.
+            - 'collections_meta', the metadata for the
+              collections.
+            - 'description', a short description of the Neurovault dataset.
+
+    Notes
+    ------
+
+    This function is only a caller for the fetch_localizer_contrasts in order
+    to simplify examples reading and understanding.
```
Comment (GaelVaroquaux) [inline nilearn/datasets/neurovault.py]: We should probably add a link to the neurovault web view of the corresponding image

File: nilearn/datasets/neurovault.py [hunk 9]
```diff
@@ -2600,3 +2600,86 @@ def fetch_neurovault_ids(
...
```
Comment (GaelVaroquaux) [inline nilearn/datasets/neurovault.py]: We should probably add a link to the neurovault web view of the corresponding image

**Comment by GaelVaroquaux** [comment]
These are all my comments: just cosmetics.

**Comment by GaelVaroquaux** [comment]
I have addressed my comments. This is good for merge once the CI is green.

**Comment by GaelVaroquaux** [comment]
I don't know why CI is failing :(

**Comment by jeromedockes** [comment]
> I don't know why CI is failing :(

One difference that might be relevant wrt other PRs is that for some reason here
ci seems to be using ubuntu 12, while for other PRs it is ubuntu 14

  CIRCLE_BUILD_IMAGE=ubuntu-12.04

gets exported, whereas for others I have checked:

  CIRCLE_BUILD_IMAGE=ubuntu-14.04

another difference with other PRs is:

$ Checkout using deploy key: 72:8a:e0:f1:a7:e2:fb:9a:71:bf:35:a6:1a:5a:38:0a

whereas others I have checked perform: 

$ Checkout using CircleCI fork PR checkout user key: 72:d2:fd:18:4c:f5:15:b1:ab:47:44:65:9d:9b:08:0a

**Comment by jeromedockes** [comment]
the job is started by organization: bthirion, whereas for other PRs it is nilearn

**Comment by GaelVaroquaux** [comment]
I'm going to do a mirrored PR of this one, to see if it passes the tests.

**Comment by bthirion** [comment]
Yes, sorry, i was not able to work on it. Thx.

**Comment by bthirion** [comment]
@jeromedockes does it mean that i should change some configuration on CircleCI ?

**Comment by jeromedockes** [comment]
> @jeromedockes does it mean that i should change some configuration on CircleCI ?

you can go to your circleCI user settings https://circleci.com/gh/organizations/bthirion/settings and see if you have anything configured for nilearn.

@KamalakerDadi and @GaelVaroquaux definitely know a lot more about CircleCi than I do

**Pull request merged**